CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
 - Class properties automatically indexed
 - Creted separated indexes for Items, Tests, Test-Takers, Groups and Deliveries
 - Added a query builder to handle searches on different indexes
+- Updated indexes when a property is changed
```

Comment:
Property of what, please be more explicit about the changes.

Reply:
Many Thanks. I made some improvements. Please, take a look :)